<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-30T14:17:44Z" name="Suite" finished-at="2022-10-30T14:20:15Z" duration-ms="151137">
    <groups>
      <group name="sanity">
        <method signature="SmartWatches.method1(java.lang.String)[pri:1, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" name="method1" class="com.facebook.stepdefaniton.SmartWatches"/>
        <method signature="SmartWatches.method2(java.lang.String)[pri:2, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" name="method2" class="com.facebook.stepdefaniton.SmartWatches"/>
        <method signature="SmartWatches.method3(java.lang.String)[pri:3, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" name="method3" class="com.facebook.stepdefaniton.SmartWatches"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2022-10-30T14:17:44Z" name="Test-1" finished-at="2022-10-30T14:20:15Z" duration-ms="151137">
      <class name="com.facebook.stepdefaniton.SmartWatches">
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:17:44Z" name="beforeClass" finished-at="2022-10-30T14:19:49Z" duration-ms="124597" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="method1(java.lang.String)[pri:1, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:19:49Z" name="method1" finished-at="2022-10-30T14:19:56Z" duration-ms="7468" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[jaya]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method2(java.lang.String)[pri:2, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:19:56Z" name="method2" finished-at="2022-10-30T14:20:05Z" duration-ms="8550" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[paul]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method2 -->
        <test-method signature="method3(java.lang.String)[pri:3, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:20:05Z" name="method3" finished-at="2022-10-30T14:20:07Z" duration-ms="2574" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[smart watches]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [paul] but found [jaya]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [paul] but found [jaya]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.facebook.stepdefaniton.SmartWatches.method3(SmartWatches.java:87)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method3 -->
        <test-method signature="method3(java.lang.String)[pri:3, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:20:08Z" name="method3" finished-at="2022-10-30T14:20:15Z" duration-ms="7698" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[smart watches]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [paul] but found [jaya]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [paul] but found [jaya]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.facebook.stepdefaniton.SmartWatches.method3(SmartWatches.java:87)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method3 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.facebook.stepdefaniton.SmartWatches@4ddbbdf8]" started-at="2022-10-30T14:20:15Z" name="afterClass" finished-at="2022-10-30T14:20:15Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.facebook.stepdefaniton.SmartWatches -->
    </test> <!-- Test-1 -->
  </suite> <!-- Suite -->
</testng-results>
